src/gui/tray/syncstatusmodel.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "folderman.h"
+
+#include <theme.h>
+#include <folder.h>
+
+#include <QObject>
+
+namespace OCC {
+
+class SyncStatusModel : public QObject
```

Comment:
please remove `Model` so as not to suggest `QAbstractItemModel` as base class

Reply:
I'm having a hard time removing it. This will produce name clashes inside Qml with SyncStatus.qml